### PR TITLE
Update manage-addins-in-the-admin-center.md

### PR DESCRIPTION
**Pull request by msbemba**
fixed deadlink for centralized deployment powershell cmdlets to manage add-ins

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4168

File: microsoft-365/admin/manage/manage-addins-in-the-admin-center.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ If the deployed add-in doesn't support add-in commands or if you want to view al
 
 Learn more about creating and building [Office Add-ins](https://docs.microsoft.com/office/dev/add-ins/overview/office-add-ins).
   
-[Use Centralized Deployment PowerShell cmdlets to manage add-ins](https://docs.microsoft.com/office365/enterprise/use-the-centralized-deployment-powershell-cmdlets-to-manage-add-ins).
+[Use Centralized Deployment PowerShell cmdlets to manage add-ins](https://docs.microsoft.com/microsoft-365/enterprise/use-the-centralized-deployment-powershell-cmdlets-to-manage-add-ins).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-addins-in-the-admin-center.md]:
```suggestion
[Use the Centralized Deployment PowerShell cmdlets to manage add-ins](https://docs.microsoft.com/microsoft-365/enterprise/use-the-centralized-deployment-powershell-cmdlets-to-manage-add-ins)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request closed without merging**